addons/mail/models/discuss/discuss_channel.py

```diff
@@ -109,6 +110,18 @@ def _compute_is_chat(self):
         for record in self:
             record.is_chat = record.channel_type == 'chat'
 
+    @api.depends('channel_type', 'is_member')
+    def _compute_is_editable(self):
+        for channel in self:
+            if channel.channel_type == 'channel':
+                channel.is_editable = self.sudo(False).env.is_admin() or channel.create_uid.id == self.env.user.id
+            elif channel.channel_type == 'chat':
+                channel.is_editable = True
```

Comment:
remove 
`
elif channel.channel_type == 'chat':
                channel.is_editable = True
`

Reply:
Okay, I get the needs, though it is a little bit strange to change the name of a Persona. but if it is the need then it is fine.

Reply:
It's the channel custom name, not the name of the persona. It's a strange "alias" feature that is not really useful to be fair... Maybe we want to have "notes" on persona like discord instead?

Reply:
-> For me I think it makes sense to have `is_editable` `false` in case of `chat`.
It's not the channel that is edited, but the custom name of the member. This should be a separate condition.

Reply:
agreed, so I suggest to update the custom name of the member if it is DM.